src/api/.haml-lint.yml

```diff
@@ -32,6 +32,10 @@ linters:
       - Style/IfUnlessModifier
       - Style/Next
       - Style/WhileUntilModifier
+    exclude: 
+      # the linter has trouble understanding the if/else of the view
+      - 'app/views/webui2/webui/user/_activity_date_commits.html.haml'
```

Comment:
there is a bug in haml-lint because it complains about something which doesn't happen. In my opinion, we should report it (we need to report it @coolo!) and exclude it until it is fixed instead of using work arounds.

Reply:
@Ana06 even if that's a fact, which you are probably right, the views have more logic than normal.. so would that possible to extract it to helpers?